### ISSUE TITLE
NullPointerException with PDF Printing

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#187]
When using Print PDF, a `NullPointerException` is thrown. 

**OS:** macOS and Windows 10 (Linux untested)
**Tray Version:** Latest branch. This is not reproducible with the stable release

```
[DEBUG] 2017-04-09 16:51:46,914 @ qz.ws.PrintSocketClient:?
	Message: {"call":"print","promise":{},"params":{"printer":{"name":"PDFwriter"},"options":{"colorType":"color","copies":"1","density":"","duplex":false,"fallbackDensity":600,"interpolation":"","jobName":"","margins":"0","orientation":"","paperThickness":"","printerTray":"","rasterize":true,"rotation":"0","scaleContent":true,"size":null,"units":"in","altPrinting":false,"encoding":"","endOfDoc":"","perSpool":"1"},"data":[{"type":"pdf","data":"file:///Applications/QZ%20Tray.app/demo/assets/pdf_sample.pdf"}]},"timestamp":1491771106913,"uid":"97o8m6"}
[WARN] 2017-04-09 16:51:46,919 @ qz.ws.PrintSocketClient:?
	Bad signature on request
[INFO] 2017-04-09 16:51:48,011 @ qz.common.TrayManager:?
	Allowed localhost to print to PDFwriter
[TRACE] 2017-04-09 16:51:48,012 @ qz.utils.PrintingUtilities:?
	Waiting for processor, 0/3 already in use
[DEBUG] 2017-04-09 16:51:48,014 @ qz.utils.PrintingUtilities:?
	Using qz.printer.action.PrintPDF to print
[DEBUG] 2017-04-09 16:51:48,014 @ qz.printer.PrintServiceMatcher:?
	Searching for PrintService matching PDFwriter
[DEBUG] 2017-04-09 16:51:48,015 @ qz.printer.PrintServiceMatcher:?
	Found 3 printers
[DEBUG] 2017-04-09 16:51:48,037 @ qz.printer.PrintServiceMatcher:?
	Found match: PDFwriter
[WARN] 2017-04-09 16:51:48,038 @ qz.printer.PrintOptions:?
	Cannot read  as a double for density, using default
[WARN] 2017-04-09 16:51:48,039 @ qz.printer.PrintOptions:?
	Cannot read  as a valid value for interpolation, using default
[WARN] 2017-04-09 16:51:48,040 @ qz.printer.PrintOptions:?
	Cannot read  as a valid value for orientation, using default
[WARN] 2017-04-09 16:51:48,041 @ qz.printer.PrintOptions:?
	Cannot read  as a double for paperThickness, using default
[DEBUG] 2017-04-09 16:51:48,052 @ qz.utils.ShellUtilities:?
	Executing: [lpoptions, -p, zebra, -l]
[DEBUG] 2017-04-09 16:51:48,088 @ qz.utils.ShellUtilities:?
	Executing: [lpoptions, -p, zebra_class, -l]
[DEBUG] 2017-04-09 16:51:48,127 @ qz.utils.ShellUtilities:?
	Executing: [lpoptions, -p, PDFwriter, -l]
[DEBUG] 2017-04-09 16:51:48,168 @ qz.utils.ShellUtilities:?
	Executing: [lpoptions, -p, Zebra_epl, -l]
[DEBUG] 2017-04-09 16:51:48,224 @ qz.printer.action.PrintPDF:?
	Parsed 1 files for printing
[ERROR] 2017-04-09 16:51:48,246 @ qz.utils.PrintingUtilities:?
	Failed to print
java.lang.NullPointerException
	at qz.printer.action.PrintPDF.applyDefaultSettings(Unknown Source)
	at qz.printer.action.PrintPDF.print(Unknown Source)
	at qz.utils.PrintingUtilities.processPrintRequest(Unknown Source)
	at qz.ws.PrintSocketClient.processMessage(Unknown Source)
	at qz.ws.PrintSocketClient.onMessage(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:70)
	at org.eclipse.jetty.websocket.common.events.annotated.OptionalSessionCallableMethod.call(OptionalSessionCallableMethod.java:68)
	at org.eclipse.jetty.websocket.common.events.JettyAnnotatedEventDriver$2.run(JettyAnnotatedEventDriver.java:210)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:745)
[TRACE] 2017-04-09 16:51:48,254 @ qz.utils.PrintingUtilities:?
	Returning processor back to pool
```

**Comment by tresf** [comment]
I believe this was introduced with https://github.com/qzind/tray/commit/76abb698faa7ae382ebda60c735afc2386e5b6fb.  

@klabarge does this still occur if you specify an orientation?

**Comment by bberenz** [comment]
Issue does occur on default orientation only, my fault, fixed in 1641e554b4cf2afbafeda1e983ad00abd044637c

**Comment by tresf** [comment]
> fixed in 1641e55

Straight forward enough, no unit testing needed. 👍 Closing.
